### ISSUE TITLE
styled_components_ts___default(...) is not a function

### ISSUE THREAD

**Issue by rr-kalbeck** [jacob-ebey/styled-components-ts#1]
hi.. i'm getting the following error..

__WEBPACK_IMPORTED_MODULE_1_styled_components_ts___default(...) is not a function

```
import styled from 'styled-components';
import styledProps from 'styled-components-ts';

interface FooStyledProps {
    bg : string;
}

const FooStyled = styledProps<FooStyledProps>( styled.div )`
    ul {
        li {
            background: ${ ( props ) => props.bg };
        }
    }
`;

export { FooStyled as default };

```

am i doing something wrong?

**Comment by nderscore** [comment]
Bump 😄 

It looks like the compiled library in `dist/` is empty when importing this package from npm?

@jacob-ebey

**Comment by andrewl913** [comment]
this is still an issue, our team is running into the same error

**Comment by jacob-ebey** [comment]
Sorry for the delay on this. This should be fixed on version 0.0.8.

**Comment by jacob-ebey** [comment]
It's the damn circleci config. Will fix tonight/tomorrow unless someone else wants to submit a PR that:

1.) adds 'persist to workspace' to the build job in the circleci config
2.) verifys the whitelist for persisting to workspace in the circleci config
3.) verifys npmignore
4.) increments npm version

**Comment by jacob-ebey** [comment]
Fixed in version 0.0.13. The new NPM package looks like:
```
- .npmignore
- LICENSE
- package.json
- README.md
- lib
    - styled-components-ts.d.ts
    - styled-components-ts.js
```
